src/components/button/Button.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const Button = ({
+  children,
+  secondary,
+  loading,
+  disabled,
+  accessibilityLabel,
+  onClick = () => { },
+  to,
+  className
+}) => {
+  const buttonClassNames = classNames(
+    'button',
+    {
+      'button--secondary': secondary,
+      'cursor-not-allowed': disabled
+    },
+    className
+  );
+
+  const onClickHandler = e => {
+    if (!loading && !disabled) {
+      onClick(e);
+    }
+  };
+
+  if (to) {
+    if (disabled) {
+      return (
+        <span aria-label={accessibilityLabel} className={buttonClassNames}>
+          {children}
+        </span>
+      );
+    }
+    return (
+      <NavLink
+        to={to}
```

Comment:
Something to try..

Put the `onClickHandler ` on here too as well as the `to` and see if it handles the disabled/loading states, rather than conditionally rendering the `span`

Reply:
It didn't like that man, passed the props though fine but didn't do anything clever in regards to disabling the anchor